staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,34 @@
+* {
```

Comment:
TIP group common css rules (mobile and desktop) in main.css

Reply:
Ok, I,ll change it, thanks!